postgres/content.md

```diff
@@ -45,6 +45,10 @@ This environment variable is recommended for you to use the PostgreSQL image. Th
 
 This optional environment variable is used in conjunction with `POSTGRES_PASSWORD` to set a user and its password. This variable will create the specified user with superuser power and a database with the same name. If it is not specified, then the default user of `postgres` will be used.
 
+### `POSTGRES_DB`
+
+This optional environment variable can be used to choose the name of your initial database. The PostgreSQL container will automatically create a database using the name provided in this variable.
```

Comment:
Thanks! Should probably mention that it defaults to the value of `POSTGRES_USER`.